lacci/lib/shoes/drawables/codes.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Shoes
+  class Codes < Shoes::Drawable
```

Comment:
That would be weird, because it's under Shoes.

Reply:
I don't get the superclass mismatch when I check out your branch. Maybe try a "git status" and see if you have any un-checked-in changes that might cause the problem?

Reply:
is that why this PR isn't using `code` as a widget (e.g.here https://github.com/scarpe-team/scarpe/blob/main/docs/static/manual.md)

Reply:
It looks like we *have* a code TextDrawable already, called "code". So yeah, this is a bit odd - one called "codes", but also a regular Drawable not a TextDrawable.

So if you declared it as Code, not Codes, you'd get the superclass error because Code is a child of Shoes::TextDrawable.

Reply:
> is that why this PR isn't using `code` as a widget (e.g.here https://github.com/scarpe-team/scarpe/blob/main/docs/static/manual.md)

yeah..

Reply:
> It looks like we _have_ a code TextDrawable already, called "code". So yeah, this is a bit odd - one called "codes", but also a regular Drawable not a TextDrawable.
> 
> So if you declared it as Code, not Codes, you'd get the superclass error because Code is a child of Shoes::TextDrawable.

should i remove this? i dont think we are using code anywhwere? in examples?

Reply:
![Screenshot_20231017_144517](https://github.com/scarpe-team/scarpe/assets/85360080/f5fa4314-3152-48fa-8603-a19fff0526cc)
this is what i get when i use code without this codes. i dont think that manual means?

Reply:
this is what `codes` does
![Screenshot_20231017_144622](https://github.com/scarpe-team/scarpe/assets/85360080/faaa8548-c452-4bc2-9f5d-fde8671e3242)

Reply:
@noahgibbs 
> should i remove this? i dont think we are using code anywhwere? in examples?

Should we remove the code text drawable, and make this PR `code` for this part of the spec?

Reply:
https://github.com/scarpe-team/scarpe/blob/main/docs/static/manual.md#codetext--shoescode

Reply:
@gintama91 btw this looks super cool :)

![image](https://github.com/scarpe-team/scarpe/assets/7865030/bcd80f25-2523-4328-92cb-ca4d4064e7cd)

Reply:
It really does look awesome :-)

It's true that we have no legacy examples using code(), only more recent ones. In Shoes3, looking at their source, it looks like code (grep for cCode) is a text type, so you *can* use it like em() inside a para(), and it's basically just monospaced like typewriter font in HTML.

If we accept this, it should definitely be renamed to replace the other code element, not add ones named codes.